regression/goto-cc-cbmc-shared-options/object-bits/default_bits.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+test.c
+--function main
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+assertion object_bits != 8: FAILURE
```

Comment:
can you add the expected results for the other assertions in here as well? Clearly it's supposed to be always 2 successes and one failure but that might be easier to see if its spelt out.

Reply:
Done.